app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,186 @@
+<resources>
+    <string name="app_name">Espanol_Vultisig</string>
+    <string name="title_activity_main">Espanol_MainActivity</string>
+    <string name="token_logo">Espanol_Token logo</string>
+    <string name="tokens">Espanol_Tokens</string>
+    <string name="chains">Espanol_Chains</string>
+    <string name="keysign">Espanol_Keysign</string>
+    <string name="keygen">Espanol_Keygen</string>
+    <string name="danger_icon">Espanol_Danger Icon</string>
+    <string name="signing_error_please_try_again">Espanol_Signing Error. Please try again.</string>
+    <string name="signing_error_please_try_again_s">Espanol_Signing Error. Please try again. %s</string>
+    <string name="bottom_warning_msg_keygen_error_screen">Espanol_Keep devices on the same WiFi Network,\n correct vault and pair devices.\n Make sure no other devices are\n running Vultisig.</string>
+    <string name="try_again">Espanol_Try again</string>
+    <string name="vault_choose_chains">Espanol_Choose Chains</string>
+    <string name="plus_icon">Espanol_Plus Icon</string>
+    <string name="vault_settings_title">Espanol_Edit Vault</string>
+    <string name="vault_settings_details_title">Espanol_Details</string>
+    <string name="vault_settings_details_subtitle">Espanol_View your vault details</string>
+    <string name="vault_settings_backup_title">Espanol_Backup</string>
+    <string name="vault_settings_backup_subtitle">Espanol_Backup your vault share</string>
```

Comment:
Please don't add translations that are not filled with correct data. Imagine how confused a user would be.

Reply:
Done